src/hotspot/share/opto/macro.cpp

```diff
@@ -651,6 +651,9 @@ bool PhaseMacroExpand::can_eliminate_allocation(PhaseIterGVN* igvn, AllocateNode
     if (res_type == NULL) {
       NOT_PRODUCT(fail_eliminate = "Neither instance or array allocation";)
       can_eliminate = false;
+    } else if (!res_type->klass_is_exact()) {
```

Comment:
Where is this code in the tip PRs?

Reply:
https://github.com/openjdk/jdk/blob/master/src/hotspot/share/opto/macro.cpp#L576

Reply:
This is the PR that added the change: https://github.com/openjdk/jdk/pull/18851

Reply:
Ah - can you please add that to the list of PRs you've backported then?

Reply:
You'll need to update the descriptions in the other PRs as well.

Reply:
I did that, perhaps at the same time that you were typing here :-)